File: AtCoder/abc211/B.kt
private fun readLn() = readLine() ?: ""
private fun readLns() = readLn().split(" ")

fun main() {
    val t = mutableListOf("H", "2B", "3B", "HR").sorted()
    val s = mutableListOf<String>()
    for (i in 0 until 4) {
        val c = readLn()
        s.add(c)
    }
    s.sort()
    when (s) {
        t -> print("Yes")
        else -> print("No")
    }
}

File: AtCoder/abc211/B.swift
private func readLn() -> String {
	return readLine() ?? ""
}

let t = ["H", "2B", "3B", "HR"].sorted()
var s = [String]()
for _ in 0 ..< 4 {
	let c = readLn()
	s.append(c)
}
s.sort()
print((s == t ? "Yes" : "No"))